src/content/docs/style-guide/writing-docs/writer-workflow/swiftype-search.mdx

```diff
@@ -0,0 +1,159 @@
+
+## Where are Swiftype results used? [#where-used]
+
+A growing number of sites and projects use our Swiftype engine to provide search results:
+
+* The New Relic help center
+* Support landing page
+* Most non-product New Relic sites (developer for related content, docs, storefront, etc.)
+
+## Swiftype crawler info [#crawler-info]
+
+The crawler says it crawls every 12 hours. The rate at which it crawls is relatively slow, which means it could take longer than that to pick up page changes. It also uses some combination of `sitemap.xml` and link collection when crawling.
+
+We can trigger a manual crawl once every 12 hours, if needed. Or, we can have Swiftype crawl specific pages when needed.
+
+The average re-index time for docs is four days.
+
+## Manually add new page to Swiftype index [#manually-index]
+
+You may want to manually add a new page to the Swiftype index when the crawl is too slow and you want the page available right away. Keep in mind that updating the idex may produce quick results for the docs site, you may experience a lag for the docs in the product UI.
+
+To manually add a new page in Swiftype:
+
+1. Ensure the page you want indexed does not have meta robots noindex set and that it's available publicly.
+2. Go to **New Relic > Content**.
+3. In the text input box that says **Filter pages**, paste the URL of the page you want indexed.
+4. If the page is not in the index, Swiftype will provide the option to **Add this URL**. Click that option to add the page to the index.
+5. Insert the URL, making sure to do the following:
+    * Use the entire URL like so: `https://docs.newrelic.com/docs/path/to/doc/`
+    * Ensure the URL has a trailing slash
+
+Swiftype should index it within a minute or so. To check if it's indexed, refresh the page and paste the URL. Note that sometimes the admin UI is cached or lags. You can ask a colleague to check, use the REST API to check, clear the cache, or do a hard refresh of the page and check again.
+
+## Manually update existing Swiftype record [#manual-index]
+
+Similar to manually adding a new page to the index, you can update an existing record if the crawl is too slow and you want the index updated right away. Use this method if you've changed a page's URL and the old URL is redirected to the new one. For an extended discussion about this, see the section [changing URLs](#url-changes) below.
+
+To manually update or reindex an existing record:
+
+1. Go to **New Relic > Content**.
+2. Paste the old URL (the one the page was last indexed with).
+3. If it's found in the index, you should see the entry for it, so select **Details**.
+4. On the record detail page, select **Reindex**.
+
+Swiftype should reindex within a minute. Note that that record will be removed from the index, along with any custom rankings you set for it, and a new record is created for the new URL that the crawler was redirected to. So, be prepared to update the result rankings.
```

Comment:
should this info be in a callout box?

Reply:
@stumblefiend thanks for the suggestion, but I'm going to keep this in the body (outside of a callout).